src/GRA.Controllers/ViewModel/MissionControl/Triggers/TriggersDetailViewModel.cs

```diff
@@ -20,6 +20,7 @@ public class TriggersDetailViewModel
         [DisplayName("Challenges and triggers the participant must have earned")]
         public ICollection<TriggerRequirement> TriggerRequirements { get; set; }
         public string BadgeRequiredList { get; set; }
+        public string BadgeAltText { get; set; }
```

Comment:
Property should have the MaxLength attribute to help check for ModelErrors and the DisplayName attribute.

Reply:
Fixed in commit 5c181001c678cd2e8be6dbe41e151628dfdee6d7